test/profile/test-storage.js

```diff
@@ -128,4 +128,28 @@ describe('open', () => {
       done();
     }());
   });
+
+  it('Saves visits.', (done) => {
+    (async function () {
+      const tempDir = tmp.dirSync({}).name;
```

Comment:
Can we figure out the Mocha `before` or `setup` function and use it?  I think we have `before` somewhere else in the tree.  (And `after`.)